### PR TITLE
Added transparency report

### PR DESCRIPTION
**Pull request by Luminaar**
Feel free to comment, if you think something is missing.

### CODE REVIEW

File: pyconcz_2016/templates/pages/transparency.html [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+
+{% block meta_title %}Transparency Report{% endblock %}
+
+{% block content %}
+<div class="transparency">
+  <div class="container text-block">
+    <h1 id="transparency-report">Transparency Report</h1>
+      <p>
+        The organization team believes that all events should be
+        accessible and welcoming to everyone, irregardless to their
```
Comment (honzajavorek) [inline pyconcz_2016/templates/pages/transparency.html]: irregardless -> regardless?

File: pyconcz_2016/templates/pages/transparency.html [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+        background. We agree that a clear and specific Code of Conduct
+        is a necessity for any event.
+        <ul>
+          <li>
+            The Code of Conduct was based on Conference anti-harassment
```
Comment (honzajavorek) [inline pyconcz_2016/templates/pages/transparency.html]: Code of Conduct -> zalinkovat stránku s CoC

File: pyconcz_2016/templates/pages/transparency.html [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            policy from Geek feminism wiki and edited to be as specific
+            to our event as possible
+          </li>
+
+          <li>
+            Czech language version of the Code was created
```
Comment (honzajavorek) [inline pyconcz_2016/templates/pages/transparency.html]: link

File: pyconcz_2016/templates/pages/transparency.html [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+        </ul>
+      </p>
+
+      <p>
+        Anonymized incidents:
+        <ul>
+          <li>
+            During one of the keynotes, inappropriate image was used in
+            a slideshow. The speaker was reminded, that a Code of
+            Conducts applies to keynotes as well. No further action was
```
Comment (honzajavorek) [inline pyconcz_2016/templates/pages/transparency.html]: Conducts -> Conduct

**Review by honzajavorek — CHANGES_REQUESTED** [review]
Just couple of nitpicks. Otherwise good!

**Comment by honzajavorek** [comment]
BTW, I forgot once again how to deploy this site, so we might need @tricoder42 to help us (or write some docs) once this is merged.

File: pyconcz_2016/templates/pages/transparency.html (hunk 1 above)
Comment (Luminaar) [reply pyconcz_2016/templates/pages/transparency.html]: Edited

File: pyconcz_2016/templates/pages/transparency.html (hunk 2 above)
Comment (Luminaar) [reply pyconcz_2016/templates/pages/transparency.html]: Edited

File: pyconcz_2016/templates/pages/transparency.html (hunk 3 above)
Comment (Luminaar) [reply pyconcz_2016/templates/pages/transparency.html]: Solved

File: pyconcz_2016/templates/pages/transparency.html (hunk 4 above)
Comment (Luminaar) [reply pyconcz_2016/templates/pages/transparency.html]: Edited

**Pull request merged**